quickstart_lists/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains 'starter lists' of packages for bootstrapping and syncing on-prem-builders.
+
+Background: Historically we bootstraped on-prem-builders by downloading all the packages in 'core'
```

Comment:
Also would be good to update the main README to point to this file and deprecate the text that talks about using the old on-prem-archive script.

Reply:
That probably should be coordinated with #212.